charts/seed-controlplane/charts/kube-apiserver-service/templates/kube-apiserver-service.yaml

```diff
@@ -6,6 +6,13 @@ metadata:
   {{if eq .Values.cloudProvider "aws" -}}
   annotations:
     service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "3600"
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "ssl"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-timeout: "5"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-interval: "30"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-healthy-threshold: "2"
```

Comment:
should we set this to 1 to attach the LB faster to the VMs?

Reply:
Will do.